internal/services/containers/kubernetes_flux_configuration_resource.go

```diff
@@ -1093,6 +1113,17 @@ func expandGitRepositoryDefinitionModel(inputList []GitRepositoryDefinitionModel
 	return &output, &configSettings, nil
 }
 
+func expandProviderType(input string) (fluxconfiguration.ProviderType, error) {
```

Comment:
It's not necessary to use this function to check the input, because there's validation func to check that.